src/main/resources/fabric.mod.json

```diff
@@ -43,7 +47,8 @@
   "custom": {
     "modmenu": {
       "links": {
-        "modmenu.discord": "https://discord.gg/xrwHKktV2d"
+        "modmenu.discord": "https://discord.gg/xrwHKktV2d",
+        "owo.wiki": "https://docs.wispforest.io/owo/features/"
```

Comment:
Shouldn't just `modmenu.wiki` work? https://github.com/TerraformersMC/ModMenu/blob/1.19.4/src/main/resources/assets/modmenu/lang/en_us.json#L85

Reply:
> Shouldn't just `modmenu.wiki` work? https://github.com/TerraformersMC/ModMenu/blob/1.19.4/src/main/resources/assets/modmenu/lang/en_us.json#L85

Sorry... I forgot about this because I previously (before this PR) translated another Minecraft Mod Lib... and there was own "Wiki" string.